### PR TITLE
[#158141513] [WIP] icon and splash screen

### PR DESCRIPTION
**Pull request by matteodesanti**
Added icons and splash screen fot both IOS and Android and fixed app name! ;-)
The Android splash screen implementation stil needs some improvement but for now should be acceptable.

TODO's:

- [ ] Android splash screen layout (see ios to compare)
- [ ] Android splash screen  to avoid the white flash
- [ ] Android notification icons should be tested on a real device
- [ ] IOS splash screen i18n

ios splash and icons

android splash and icons

### CODE REVIEW

**Comment by cloudify** [comment]
Thanks @matteodesanti I'll have a look - btw I found [this tutorial](https://facebook.github.io/react-native/blog/2018/01/18/implementing-twitters-app-loading-animation-in-react-native) it would be awesome for the opening screen (iOS only)

File: ios/ItaliaApp/Images.xcassets/logo agid.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (cloudify) [inline ios/ItaliaApp/Images.xcassets/logo agid.imageset/Contents.json]: better avoid spaces in path (`logo agid.imageset`)

**Comment by matteodesanti** [comment]
Wow supercool!
I've completed all the steps explained in [this tutorial](https://medium.com/@scottianstewart/react-native-add-app-icons-and-launch-screens-onto-ios-and-android-apps-3bfbc20b7d4c).
Then I've started to follow [this one](https://medium.com/handlebar-labs/how-to-add-a-splash-screen-to-a-react-native-app-ios-and-android-30a3cec835ae) but i've stoppped when the author suggest to installa the `react-native-splash-screen` module, mostly because he provides examples in javascript and not in typescript. Probably a "real developer" and not a designer is needed here! ;-)

**Comment by cloudify** [comment]
@matteodesanti yes indeed, if we have all the needed assets, I can configure `react-native-splash-screen`

**Comment by cloudify** [comment]
Closing, superseded by https://github.com/teamdigitale/italia-app/pull/416

**Pull request closed without merging**